components/dropdown/Dropdown.tsx

```diff
@@ -97,10 +100,13 @@ const Dropdown: React.FC<DropdownProps> = ({
       {showButton && (
         <button
           type="submit"
-          onClick={clearAllFilters}
+          onClick={(e) => {
+            e.preventDefault();
```

Comment:
I added this `preventDefault` so the page only refreshes once (to show the search results) 
Without preventDefault it refreshed twice: 

1. Submit a form (refresh by default)
2. Get search results (another reload)